pandas/core/arrays/_mixins.py

```diff
@@ -293,7 +293,9 @@ def _fill_mask_inplace(
         func(self._ndarray.T, limit=limit, mask=mask.T)
 
     @doc(ExtensionArray.fillna)
-    def fillna(self, value=None, method=None, limit: int | None = None) -> Self:
+    def fillna(
+        self, value=None, method=None, limit: int | None = None, copy: bool = True
```

Comment:
Why copy and not inplace?

Reply:
either way.  copy seemed more future-proof

Reply:
Might be better to wait for PDEP 7/8?

Reply:
that seems stuck in the mud and this is needed to unblock other work, so id rather not wait that long.  if using inplace instead of copy makes a difference for you ill be happy to change it

Reply:
Fwiw I prefer copy here. This isn’t really user facing anyway